### ISSUE TITLE
Just taking of your cookies session (NOT HACK)

### ISSUE THREAD

**Issue by yrifl** [pashayogi/Fb-hack#6]
The script is not for hacking, its just taking Facebook cookies from the browser

Cookies : session of your login detail storage! Your facebook email and password in storage of cookies🤪

**Comment by yrifl** [comment]
""""
def main():
        global noobs
        noobs = mechanize.Browser()
        cj = cookielib.LWPCookieJar()
        noobs.set_handle_robots(False)
        noobs.set_handle_redirect(True)
        noobs.set_cookiejar(cj)
        noobs.set_handle_equiv(True)
        noobs.set_handle_referer(True)
        noobs.set_handle_refresh(mechanize._http.HTTPRefreshProcessor(), max_time=1)
        runn_noobs()
        life()
        print " "
        print RR+" wordlist tidak ada yg cocok..."
        print " "
"""

**Comment by ccrsxx** [comment]
Of course, it is, what a dumb repository. I wonder why tf GitHub keeps recommending these trash repositories.

**Comment by TrollSkull** [comment]
Yes, always check what are you downloading.

```python
def main():
  global noobs
  noobs = mechanize.Browser()
  cj = cookielib.LWPCookieJar()
  noobs.set_handle_robots(False)
  noobs.set_handle_redirect(True)
  noobs.set_cookiejar(cj)
  noobs.set_handle_equiv(True)
  noobs.set_handle_referer(True)
  noobs.set_handle_refresh(mechanize._http.HTTPRefreshProcessor(), max_time=1)
  runn_noobs()
  life()
  print " "
  print RR+" wordlist tidak ada yg cocok..."
  print " "
```

**Comment by Rajmallic** [comment]
Hi
